run_with_lambda.py

```
@@ -0,0 +1,301 @@
+            check_interval: Seconds between checks
+        """
+        self.check_interval = check_interval
+        self.filing_tracker = FilingTracker(Path("production_filings.json"))
+        self.petition_processor = Chapter11PetitionProcessor()
+        self.integration = LambdaRSSIntegration(
+            filing_tracker=self.filing_tracker,
+            petition_processor=self.petition_processor
+        )
+        self.is_running = False
+        self.stats = {
+            'start_time': datetime.utcnow(),
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)